packages/stable/src/api/annotations/annotationsApi.ts

```diff
@@ -45,13 +47,21 @@ export class AnnotationsAPI extends BaseResourceAPI<AnnotationModel> {
    * [Retrieve](https://docs.cognite.com/api/playground/#operation/annotationsByids)
    * or [Get an](https://docs.cognite.com/api/playground/#operation/annotationsGet)
    * annotation
+   *
+   * const response = await client.annotationsApi.retrieve(createdAnnotationIds);
```

Comment:
```suggestion
   * const response = await client.annotations.retrieve(createdAnnotationIds);
```
all of these are invalid..
have you tried running those in some test project?

Reply:
we cannot have code snippets that are syntactically incorrect